apps/ccf-eui/src/app/app.component.ts

```diff
@@ -335,7 +335,7 @@ export class AppComponent implements OnInit {
    * Gets login token
    */
   get loggedIn(): boolean {
```

Comment:
This won't be possible/logical anymore. It'll just have to be set from outside (if at all)